packages/snippet-bot/test/snippet-bot.ts

```diff
@@ -52,7 +52,7 @@ describe('snippet-bot', () => {
   let invalidateCacheStub: sinon.SinonStub;
 
   beforeEach(() => {
-    probot = createProbot({
```

Comment:
You can leave this as createProbot, just add `overrides`: {github token...}

Reply:
`createProbot` is deprecated anyways.